encoding.bs

```diff
@@ -2347,14 +2350,162 @@ consumers of content generated with <a>GBK</a>'s <a for=/>encoder</a>.
      <li><p>Return <a>error</a>.
     </ol>
 
-   <li><p>Let <var>code point</var> be the <a>index gb18030 ranges code point</a> for
+   <li><p>Let <var>code point</var> be null.
+
+   <li>
+    <p>If <a for="gb18030 decoder">is GBK</a> is false and there is a row in the table below whose
+    first column is <a>gb18030 first</a>, second column is <a>gb18030 second</a>, third column is
+    <a>gb18030 third</a>, and fourth column is <var>byte</var>, then set <var>code point</var> to
+    the fifth column on the same row:
```

Comment:
Based on test results in Firefox I calculated the first override here and it appears that this is already the existing behavior of gb18030 as written in the standard. As such this table isn't needed. It's not clear to me why the Unicode recommendation calls out these four-byte sequences mapping to code points if that matches existing behavior.